lighthouse-cli/test/cli/run-test.js

```diff
@@ -25,7 +25,8 @@ describe('CLI run', function() {
   it('runLighthouse completes a LH round trip', () => {
     const url = 'chrome://version';
     const filename = path.join(process.cwd(), 'run.ts.results.json');
-    const flags = getFlags(`--output=json --output-path=${filename} ${url}`);
+    const timeoutFlag = `--max-wait-for-load=${9000}`;
```

Comment:
I'm curious, why put 9000 in template? :)

Reply:
for the funsies.  🕺